CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ### Unreleased
+ - Upgraded spotless version
+ - Added vscode launch.json
```

Comment:
Shouldn't these be below the "changed" heading?

Reply:
@kingthorin yes, i will update the change log.